ifrau-client.js

```diff
@@ -0,0 +1,3 @@
+module.exports = import('ifrau/client/slim.js').then(function(ifrau) {
```

Comment:
@omsmith, this now uses the more restrictive entry-point. I gave this a test run as well and can confirm that this syntax looks good and the slim client does load. :)